src/components/sections/Footer.tsx

```diff
@@ -97,10 +98,12 @@ export default function Footer() {
 						>
 							Σ-Motion
 						</Link>{" "}
-						by
+						by Nova Mondal, Jason Ballinger, & Nathan Gillespie
```

Comment:
remove, we aren't hardcoding, see line 86

Reply:
It's commented out, I will work on the list normally soon